src/scripts/tweaks/show_all_tags.js

```diff
@@ -0,0 +1,20 @@
+import { keyToCss, resolveExpressions } from '../../util/css_map.js';
+import { buildStyle } from '../../util/interface.js';
+
+const styleElement = buildStyle();
+resolveExpressions`
+${keyToCss('tags')}${keyToCss('collapsed')} {
+  max-height: inherit;
```

Comment:
According to Firefox devtools, the default value for `max-height` is `none`, which I think is more intuitive to communicate what this CSS is doing anyhow:

```suggestion
  max-height: none;
```

Reply:
Oh yeah, makes sense!